<launch>
	<rosparam file="$(find RMC_1_hardware_interface)/config/hardware.yaml" command="load"/>
	<rosparam file="$(find RMC_1_hardware_interface)/config/controllers.yaml" command="load"/>
	<rosparam file="$(find RMC_1_hardware_interface)/config/joint_limits.yaml" command="load"/>

	<node name="RMC_1_hardware_interface" pkg="RMC_1_hardware_interface" type="RMC_1_hardware_interface_node" output="screen" />
	<node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/" 
	args="
		/ROBOT/controller/state
		/ROBOT/controller/position/steer_joint_1
		/ROBOT/controller/velocity/wheel_joint_1
	"/>
</launch>

